tests/protocols/test_websocket.py

```diff
@@ -457,3 +458,53 @@ async def get_subprotocol(url):
     async with run_server(config):
         accepted_subprotocol = await get_subprotocol("ws://127.0.0.1:8000")
         assert accepted_subprotocol == subprotocol
+
+
+MAX_WS_BYTES_PLUS1 = 1024 * 1024 * 16 + 1
+MAX_WS_BYTES = 1024 * 1024 * 16
```

Comment:
If you want, if not it's fine:
```suggestion
MAX_WS_BYTES = 1024 * 1024 * 16
MAX_WS_BYTES_PLUS1 = MAX_WS_BYTES + 1
```